assets/app/view/game/game_info.rb

```diff
@@ -318,6 +318,27 @@ def trains
         ]
       end
 
+      def power
+        props = {
+          style: {
+            textAlign: 'center',
+            width: 'max-content',
+          },
+        }
+
+        [
+          h(:h3, 'Power Progress'),
+          h(:div, { style: { overflowX: 'auto' } }, [
+            h(:table, [
+              h(:tbody, [
+                h(:tr, props, Array.new(15) { |p| h(:td, p + 1) }),
```

Comment:
what is 15

Reply:
A magic number which I need to turn into  game constant.